jina/orchestrate/pods/__init__.py

```diff
@@ -149,7 +149,7 @@ def __init__(self, args: 'argparse.Namespace'):
             getattr(args, 'runtime_backend', RuntimeBackendType.THREAD),
             events_list=[self.is_ready, self.is_shutdown],
         )
-        self.daemon = self.args.daemon
+        self.daemon = True
```

Comment:
how is this used? can u set it to true in the `worker`?

Reply:
it should be deleted